apps/flows/src/app/transaction-details-flow/for-you/for-you.component.scss

```diff
@@ -68,14 +68,14 @@ $gap: utils.size('m');
 
 .inside-kirby-card-grid {
   display: grid;
-  grid-template-columns: 300px 1fr;
-  grid-template-rows: 30px 30px;
-  gap: 5px;
+  grid-template-columns: 1fr 1fr;
```

Comment:
Alternatively you can let the size of the content itself control the column width by declaring `grid-template-columns: auto 1fr` (I think). But this is fine if two columns of equal width look ok.